libs/code-demos/src/lib/multitab-editor/multitab-editor.component.html

```diff
@@ -1,42 +1,57 @@
-<div
-  class="wrapper"
-  *ngFor="let model of openModels; trackBy: trackByEditorIndex; let i = index"
->
-  <div class="panel" *ngIf="displayFileName">
-    <code-demo-file-path
-      style="margin: 1px 0 10px 0;display: block;"
-      [path]="openModels[i].path"
-    ></code-demo-file-path>
-  </div>
-  <div class="panel" *ngIf="allowSwitchingFiles">
-    <mat-form-field>
-      <mat-select
-        [ngModel]="openModels[i]"
-        (selectionChange)="handleFileChange(i, $event)"
-      >
-        <mat-select-trigger>
-          <code-demo-file-path
-            [path]="openModels[i].path"
-          ></code-demo-file-path>
-        </mat-select-trigger>
+<mat-sidenav-container class="example-container" style="height: 100%">
```

Comment:
example-container?

Reply:
ah the class isn't used anymore I'll clean it up